lib/features/manage_account/presentation/profiles/identities/utils/identity_utils.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:jmap_dart_client/jmap/core/unsigned_int.dart';
+import 'package:jmap_dart_client/jmap/identities/identity.dart';
+
+class IdentityUtils {
+  
+  static Identity? getSmallestOrderedIdentity(List<Identity>? identities) {
```

Comment:
I am not fan of static function, how about create singleton `IdentityUtils` and use it as a method